libvast/native-plugins/rebuild.cpp

```diff
@@ -454,10 +454,15 @@ struct rebuilder_state {
             [finish](caf::error error) mutable {
               finish(std::move(error));
             });
-          VAST_INFO(
-            "{} triggered {} for {} candidate partitions with {} threads",
-            *self, run->options.automatic ? "an automatic run" : "a run",
-            run->statistics.num_total, run->options.parallel);
+          if (run->options.automatic)
```

Comment:
I think we should apply the same logic to the "had nothing to do" message that we recently demoted to verbose.